src/ecpair.js

```diff
@@ -66,11 +66,12 @@ ECPair.fromWIF = function (string, network) {
 
     network = network.filter(function (network) {
       return version === network.wif
-    }).pop() || {}
+    }).pop()
+    if (!network) throw new Error('Invalid network version')
```

Comment:
This isn't tested.
I'll make a new PR if relevant.

Thanks for the PR @fanatid :)